src/main/java/com/kuzmins/dao/UserDAO.java

```diff
@@ -0,0 +1,59 @@
+package com.kuzmins.dao;
+
+import com.kuzmins.config.ConnectionConfig;
+import com.kuzmins.model.User;
+
+import java.sql.*;
```

Comment:
do not use squashed imports. import each class separately, it can be configured int Intellij Idea settings

Reply:
fixed